source/core/shape.cc

```diff
@@ -366,6 +371,148 @@ Sphere<dim>::set_position(const Point<dim> &position)
 #endif
 }
 
+template <int dim>
+std::tuple<double, Tensor<1, dim>, Point<dim>>
+Sphere<dim>::distance_to_shape_with_cell_guess(
+  Shape<dim>                                           &shape,
+  const typename DoFHandler<dim>::active_cell_iterator &cell,
+  std::vector<Point<dim>>                              &candidate_points,
+  double                                                precision,
+  bool exact_distance_outside_of_contact)
+{
+  (void)candidate_points;
+  (void)precision;
+  (void)exact_distance_outside_of_contact;
```

Comment:
You are doing this to avoid warnings right?
If so, add a comment  above to say this.